### DISCUSSION TITLE
new functionality for business calculator and upgrading old funtionality for business calculator discussion

### DISCUSSION THREAD

**Discussion by pranav627** [pranav627/BusinessCalculator#5] [Q&A] [answered]
The purpose of this discussion is to explore new functionalities for a business calculator and discuss potential upgrades for existing features. A business calculator is a valuable tool for performing various financial calculations, such as interest rate calculations, loan amortization, profit margins, and more. By enhancing and upgrading its functionalities, we can improve its efficiency, accuracy, and overall usefulness for businesses.

**Comment by shilpa-k-h** [comment, marked as answer]
need to add multiple fields functionality in expenses tab so that you can add multiple expenses in a business

**Comment by shilpa-k-h** [comment]
adding built in currency conversion and tax percentage calculator will help in accessibility of this project
